### PR TITLE
Remove Radium from ScrollableList and ItemLoopToggle (Animation Tab elements)

### PR DESCRIPTION
**Pull request by mikeharv**
This is part of the work to stop using Radium. See https://docs.google.com/spreadsheets/d/1chviZU39Qej9_xrc2lrcRrydKJKcs-W6Q0ehzVtH6bg/edit#gid=1128290845 for all components using Radium.
This PR includes changes to two animation tab components: ScrollableList and ItemLoopToggle.

Scrollable List:

ItemLoopToggle:

## Testing story

No regressions found. Compared styles for each styled element in the DOM.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/p5lab/AnimationTab/ItemLoopToggle.jsx [hunk 1]
```diff
@@ -30,21 +30,15 @@ class ItemLoopToggle extends React.Component {
     return (
       <OverlayTrigger overlay={tooltip} placement="bottom" delayShow={500}>
         <div
-          style={[styles.loopToggleStyle, this.props.style]}
+          style={{...this.props.style}}
+          className={style.loopToggle}
           onClick={this.toggleClicked}
         >
-          <img src={iconImageSrc} style={styles.loopIconStyle} />
+          <img src={iconImageSrc} className={style.loopIconStyle} />
```
Comment (bencodeorg) [inline apps/src/p5lab/AnimationTab/ItemLoopToggle.jsx]: I don't see `loopIconStyle` anywhere (in previous version or new version), am I missing something?

File: apps/src/p5lab/AnimationTab/ItemLoopToggle.jsx [hunk 2]
```diff
@@ -30,21 +30,15 @@ class ItemLoopToggle extends React.Component {
     return (
       <OverlayTrigger overlay={tooltip} placement="bottom" delayShow={500}>
         <div
-          style={[styles.loopToggleStyle, this.props.style]}
+          style={{...this.props.style}}
```
Comment (bencodeorg) [inline apps/src/p5lab/AnimationTab/ItemLoopToggle.jsx]: Nit: `style={this.props.style}`

File: apps/src/p5lab/AnimationTab/scrollable-list.module.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.root {
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+.margins {
+  margin: 4;
```
Comment (bencodeorg) [inline apps/src/p5lab/AnimationTab/scrollable-list.module.scss]: Nit: 4px? Does this work without specifying units?

**Comment by bencodeorg** [comment]
Maybe update PR title to describe what components you're removing Radium from?

**Comment by mikeharv** [comment]
> Maybe update PR title to describe what components you're removing Radium from?

Whoops, great call! :)

File: apps/src/p5lab/AnimationTab/ItemLoopToggle.jsx (hunk 1 above)
Comment (mikeharv) [reply apps/src/p5lab/AnimationTab/ItemLoopToggle.jsx]: Huh, good point. I think I was on autopilot for this change and missed that. Seems safe enough to just remove this.

File: apps/src/p5lab/AnimationTab/scrollable-list.module.scss (hunk 3 above)
Comment (mikeharv) [reply apps/src/p5lab/AnimationTab/scrollable-list.module.scss]: It most certainly did not, but it should now!

File: apps/src/p5lab/AnimationTab/ScrollableList.jsx [hunk 4]
```diff
@@ -28,14 +19,14 @@ class ScrollableList extends React.Component {
   render() {
     return (
       <div
-        className={this.props.className}
-        style={[staticStyles.root, this.props.style]}
+        className={classNames(this.props.className, style.root)}
+        style={{...this.props.style}}
```
Comment (bencodeorg) [inline apps/src/p5lab/AnimationTab/ScrollableList.jsx]: Sorry one more destructuring nit I missed

**Review by bencodeorg — APPROVED** [review]

**Pull request merged**